backend/app.py

```
@@ -84,7 +128,13 @@ def seed():
 
 
 # main guard
-# port and host changed for deployment purposes
 if __name__ == "__main__":
     app = create_app()
-    app.run(debug=True, host="0.0.0.0", port=5000)
+    if app.config["ENV"] == "production":
+        logger.debug("Running in production mode")
+        # Use gunicorn in production, don't run app.run()
+        pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)